packages/protocol/test/L1/TaikoL1.test.ts

```diff
@@ -104,6 +157,328 @@ describe("TaikoL1", function () {
     });
 });
 
+describe("integration:TaikoL1", function () {
+    let taikoL1: TaikoL1;
+    let taikoL2: TaikoL2;
+    let l2Provider: ethersLib.providers.JsonRpcProvider;
+    let l2Signer: ethersLib.Signer;
+
+    beforeEach(async function () {
+        l2Provider = new ethers.providers.JsonRpcProvider(
+            "http://localhost:28545"
+        );
+
+        l2Signer = await l2Provider.getSigner(
+            "0x4D9E82AC620246f6782EAaBaC3E3c86895f3f0F8"
```

Comment:
```suggestion
            (await l2Provider.listAccounts())[0]
```
Lets get the signer account address from the `listAccounts`? instead of hardcoding.